src/overview/components/Overview.jsx

```diff
@@ -75,6 +91,9 @@ const mapDispatchToProps = dispatch => ({
     onEndDateChange: date => {
         dispatch(actions.setEndDate({endDate: date}))
     },
+    getMoreResults: (date, position) => {
+        dispatch(actions.getMoreResults({loadingIndicator: true, endDate: date, scrollPosition: position}))
```

Comment:
This was a little issue as... the state of waiting for results is needed when trigering the fetch results which is currently only changed by the showloading and hide action, so i did not want to compromise that standard.